### ISSUE TITLE
Left shift and caps lock don't work with script running

### ISSUE THREAD

**Issue by mdwebster** [YosfanEilay/Fix-keyboard-DDDouble-Tyypingg#1]
Found this and it seems to work well to prevent double key presses on a G613, but I noticed the left shift doesn't work to make capital letters.  Right shift does.  Left and right ctrl work, left and right alt works.  Messing around, I discovered Caps Lock also doesn't work.  Running AHK 1.1 on Windows 11.  G613 is attached via bluetooth so no GHub involvement.

Best regards ...

**Comment by YosfanEilay** [comment]
Hello @mdwebster, thanks for your feedback, will work on it when i have time to fix it and will release a new version of this fix,
is that effecting your day to day job? or just bugging you?

**Comment by mdwebster** [comment]
No, you're good.  I only use this keyboard at home.  I just thought I'd report it.

**Comment by YosfanEilay** [comment]
@mdwebster cool, thanks again for reporting, will work on it when i have free time ^_^

**Comment by nobleodin** [comment]
thank you very much for this script - I too am expericing the same issue as reported above.  Would be grateful if a fix could be provided.

**Comment by antaig** [comment]
This script seems like a lifesaver for my aging G613, you're doing some amazing work.

I have to report that I still have the issue stated above, is there a way to personally tweak the script in order to take care of it?

**Comment by YosfanEilay** [comment]
Hi @antaig Thanks for the support!
I haven’t had time to work on it yet, as I’ve been focused on my work repositories. This is a personal project, not work-related, so I only work on it when I have free time. However, it’s written in AutoHotkey, which is easier to tweak. Feel free to experiment with the code, and you can even use ChatGPT to request changes or adjustments.

**Comment by Rhavyc** [comment]
Ok, I do use this for work, so I am going to need to know how to edit this, please. I've been able to get around it so far but now it's becoming an issue.

**Comment by Eronponce** [comment]
> Ok, I do use this for work, so I am going to need to know how to edit this, please. I've been able to get around it so far but now it's becoming an issue.


Check my code 
https://github.com/Eronponce/G613-Keyboard-double-type-Fix
